eclipse/che
# Propagate CA certs bundle on all infrastructures

Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Propagates CA bundle into user's workspaces on all infrastructures.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17634

### How to test this PR?
1. Deploy Che on a Kubernetes family infrastructure
2. Create a config map with CA certificates in `che` namespace
3. Add the config map into CR:
```yaml
spec:
  server:
    serverTrustStoreConfigMapName: custom-ca
```
3. Start a workspace, check that `/public-certs` has the certificate in it.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
